package.json

```
@@ -90,6 +90,7 @@
     "lint:md": "remark content/posts/",
     "write-good": "write-good $(glob 'content/posts/**/*.md')",
     "format:js": "prettier '**/*.{js,jsx}' --write",
+    "format-check:js": "prettier '**/*.{js,jsx}' --check",
```

Ignore the same stuff as eslint, otherwise we scan caches and output folders and such.

```suggestion
    "format-check:js": "prettier '**/*.{js,jsx}' --check --ignore-path=.eslintignore",
```